### PR TITLE
weakref.__reversed__ fixed

### PR DESCRIPTION
**Pull request by jinnarajin**
rebase

### CODE REVIEW

File: vm/src/builtins/weakproxy.rs [hunk 1]
```diff
@@ -4,11 +4,17 @@ use crate::{
     class::PyClassImpl,
     common::hash::PyHash,
     function::{OptionalArg, PyComparisonValue, PySetterValue},
+<<<<<<< HEAD
```
Comment (DimitrisJim) [inline vm/src/builtins/weakproxy.rs:7]: There's some conflicts here that need cleaning up :-)

Comment (fanninpm) [reply vm/src/builtins/weakproxy.rs:7]: I was about to say… @jinnarajin  have you tried compiling this locally?

**Comment by DimitrisJim** [comment]
Closing in favor of https://github.com/RustPython/RustPython/pull/4780/

File: vm/src/builtins/weakproxy.rs (hunk 1 above)
Comment (jinnarajin) [reply vm/src/builtins/weakproxy.rs:7]: No...

**Pull request closed without merging**